### DISCUSSION TITLE
Vulnerablity Feed Question

### DISCUSSION THREAD

**Discussion by Wyattmontana** [Lissy93/dashy#609] [Help and Support] [answered]
Hi
I was wondering what option it is to toggle the summarys on the CVE widget?
![image](https://user-images.githubusercontent.com/47372462/164817467-8f5ee0c2-7fe7-41cd-bd56-2563b44d9606.png)

I have tried a few things but mine only does the CVE ID and score
![image](https://user-images.githubusercontent.com/47372462/164817606-2e9bb949-da13-457d-aa20-093b5e0d2a27.png)
Any insight is appreciated!

**Comment by lissy93** [comment]
Hmm looks like there could have been a change to the API, I'll look into it in the morning...

**Reply by Wyattmontana** [reply]
Thank you!

**Reply by lissy93** [reply]
Oops, I actually forgot about this ha ha
I've just had a look, and turns out it was a dumb CSS error on my part
I just pushed a fix out in 45fa35a384d3d9228e2f2396b2b5b3ede66d6e48 so description should now be visible again
Thanks for your patience  :)

**Reply by Wyattmontana** [reply]
Thank you so much for the fix! I really appreciate it! :)

**Comment by lissy93** [comment, marked as answer]
✅ Fixed in 45fa35a384d3d9228e2f2396b2b5b3ede66d6e48
